bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -310,8 +329,11 @@ func (s *ksServer) CreateApp(ctx context.Context, request CreateRequest) error {
 	} else {
 		log.Infof("Creating app %v", request.Name)
 		log.Infof("Using K8s host %v", config.Host)
-
-		appDir := path.Join(s.appsDir, GetRepoName(request.Project), request.Name)
+		deployConfDir := path.Join(s.appsDir, GetRepoName(request.Project), kfVersion, request.Name)
```

Comment:
What is s.appsDir? Is that a directory in the volume where things are stored?

Reply:
Yes, that's where we checkout source repos